_posts/2015-05-13-edition-3.markdown

```diff
@@ -141,7 +141,7 @@ Sébastien's work is very welcome.
 
 Unfortunately git developers can have different views on how to group
 commands together. So it can be difficult for them to agree on such
-kind of changes. Long discussions because of small personnal
+kind of changes. Long discussions because of small personnel
```

Comment:
I think this should actually be "personal". Sorry for not spotting this before.

Reply:
Otherwise everything looks good, thanks! Let me know if you want me to fix this.

Reply:
Absolutely, thanks!

BTW, thanks to your comment I've added an extra one: bikesheedding -> bikeshedding.